scenario-compiler/src/test/scala/pl/touk/nussknacker/engine/spel/SpelExpressionSpec.scala

```diff
@@ -2056,6 +2060,51 @@ class SpelExpressionSpec extends AnyFunSuite with Matchers with ValidatedValuesD
     parsed.evaluateSync[Any](customCtx) shouldBe 11
     parsed.evaluateSync[Any](customCtx) shouldBe 11
   }
+  /*
```

Comment:
Arek found the root issue with expression evaluation so we modified `getValue` accordingly. Since the method is even a bit more complex now we agreed to add this test to ignored.